src/NUnitFramework/Directory.Build.props

```diff
@@ -8,7 +8,7 @@
     <DisableImplicitNuGetFallbackFolder>true</DisableImplicitNuGetFallbackFolder>
     <EnforceCodeStyleInBuild>true</EnforceCodeStyleInBuild>
     <NUnitLibraryFrameworks>net462;net6.0</NUnitLibraryFrameworks>
-    <NUnitRuntimeFrameworks>net462;net6.0;net7.0</NUnitRuntimeFrameworks>
+    <NUnitRuntimeFrameworks>net462;net6.0;net7.0;net8.0</NUnitRuntimeFrameworks>
```

Comment:
Should we add net8.0 to NUnitLibraryFramework so we have a native build for this or not?

Reply:
I agree, I was going to add that, but it was too much typing on the mobile.